### PR TITLE
Add Dutch translations

### PR DESCRIPTION
**Pull request by DCSBL**

### CODE REVIEW

File: custom_components/huesyncbox/translations/nl.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      "intensity_intense": "Stel intensiteit in op extreem",
+      "sync_on": "Start syncronisatie",
+      "sync_off": "Stop syncronisatie",
+      "sync_toggle": "Schakel syncronisatie aan/uit",
+      "sync_video_toggle": "Schakel video mode syncronisatie",
+      "sync_music_toggle": "Schakel music mode syncronisatie",
+      "sync_game_toggle": "Schakel game mode syncronisatie",
+      "sync_video_on": "Start video mode syncronisatie",
+      "sync_music_on": "Start music mode syncronisatie",
+      "sync_game_on": "Start game mode syncronisatie",
```
Comment (mvdwetering) [inline custom_components/huesyncbox/translations/nl.json:63]: Should be "synchronisatie" (missing `h`) for all occurrences

File: custom_components/huesyncbox/translations/nl.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      "input_previous": "Selecteer vorige input",
+      "input_hdmi1": "Selecteer HDMI1",
+      "input_hdmi2": "Selecteer HDMI2",
+      "input_hdmi3": "Selecteer HDMI3",
+      "input_hdmi4": "Selecteer HDMI4",
+      "mode_next": "Selecteer volgende modus",
+      "mode_previous": "Selecteer vorige modus",
+      "mode_game": "Stel modes in op game",
+      "mode_music": "Stel modes in op music",
+      "mode_video": "Stel modes in op video",
```
Comment (mvdwetering) [inline custom_components/huesyncbox/translations/nl.json:74]: Should be "mode" instead of modes

**Review by mvdwetering — CHANGES_REQUESTED** [review]
Thanks for the translation. I left a few minor comments

File: custom_components/huesyncbox/translations/nl.json (hunk 1 above)
Comment (DCSBL) [reply custom_components/huesyncbox/translations/nl.json]: 🤦‍♂️

**Review by mvdwetering — APPROVED** [review]

**Pull request merged**